src/confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -21,6 +21,8 @@
 #
 import logging
 import warnings
+import urllib3
```

Comment:
For now, its fine to use both urllib3 and requests. Create a backlog to move everything from requests to urllib3.

Reply:
Sure